backend/src/routes/v2/model/postModel.ts

```diff
@@ -4,14 +4,15 @@ import { z } from 'zod'
 
 import { AuditInfo } from '../../../connectors/audit/Base.js'
 import audit from '../../../connectors/audit/index.js'
-import { EntryVisibility, ModelInterface } from '../../../models/Model.js'
+import { EntryKind, EntryVisibility, ModelInterface } from '../../../models/Model.js'
 import { createModel } from '../../../services/model.js'
 import { modelInterfaceSchema, registerPath } from '../../../services/specification.js'
 import { parse } from '../../../utils/validate.js'
 
 export const postModelSchema = z.object({
   body: z.object({
     name: z.string(),
+    kind: z.nativeEnum(EntryKind).optional().default(EntryKind.Model),
```

Comment:
Should we force this to be specified so that API users can't accidentally create a model when they intended to make a data card?